combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt

```diff
@@ -264,7 +264,6 @@ internal class NetworkManager(
             serialNumber = probe.probeSerialNumber,
             owner = owner,
             settings = settings,
-            dfuConnectedNodeCallback = { },
```

Comment:
This, oddly, wasn't used. I'm guessing we never found a way to integrate the logic with the firmware version coming later.

Reply:
🤷🏻